FreydCategoriesForCAP/gap/RingsAsAbCats.gi

```diff
@@ -302,6 +323,98 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_RING_AS_CATEGORY,
             
         end );
         
+        ##
+        AddTensorUnit( category,
+          function( cat )
+            
+            return RingAsCategoryUniqueObject( cat );
+            
+        end );
+        
+        ##
+        AddTensorProductOnObjects( category,
+          function( cat, a, b )
+            
+            return RingAsCategoryUniqueObject( cat );
+            
+        end );
+        
+        ##
+        AddTensorProductOnMorphismsWithGivenTensorProducts( category,
+          function( cat, source, alpha, beta, range )
+            
+            return MorphismConstructor( cat,
```

Comment:
Please use `RingAsCategoryMorphism` to stay consistent to the existing code (e.g. in `MultiplyWithElementOfCommutativeRingForMorphisms`).

Reply:
Done.